### PR TITLE
add git auto-update config of jquery.cycle, close #5699

### LINKED ISSUE #5699: Auto-update config of jquery.cycle
https://cdnjs.com/libraries/jquery.cycle
Repo: https://github.com/malsup/cycle

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/malsup/cycle
**Watch 101
  Star 1,233
 Fork 300**
@Amomo   could u help me check this PR for #5699 ? thank you!
- [ ] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/jquery.cycle/package.json [hunk 1]
```diff
@@ -19,5 +19,17 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/malsup/cycle.git",
+    "basePath": "",
+    "files": [
+      "jquery.cycle*.js"
+    ]
+  },
+  "requiredFiles": [
+    "jquery.cycle.all.min.js",
```
Comment (Amomo) [inline ajax/libs/jquery.cycle/package.json]: filename field already point to `"jquery.cycle.all.min.js"`

**Comment by maruilian11** [comment]
@Amomo could you help me check the PR again? thank you!

**Comment by maruilian11** [comment]
@Amomo could you help me check the PR again? thank you!

**Comment by maruilian11** [comment]
@Amomo could you help me check the PR again? thank you!

**Comment by Amomo** [comment]
Hi @maruilian11 I am so sorry to keep you waiting so long.😰
I think this PR is fine, could you rebase it to match latest main branch?
Thank you! :-D

**Comment by maruilian11** [comment]
@Amomo i have rebased it~ thank you :joy:

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you! :-D

**Pull request merged**